products/posters.js

```diff
@@ -0,0 +1,109 @@
+// export default
+//(Remember: the Add button is = product id!) 
+
```

Comment:
Notes are great but also try to keep your code at the top of the file and remove any comments you no longer need.

Reply:
Sounds good